scripts/release.sh

```diff
@@ -50,7 +50,7 @@ SCRIPTDIR=$(dirname "$0")
 TAG=`get_tag`
 BRANCH=`git rev-parse --abbrev-ref HEAD`
 IMAGES=
-DEFAULT_IMAGES="exporters.metrics.pool"
+DEFAULT_IMAGES="exporters.metrics.pool operators.upgrade obs.callhome"
```

Comment:
Please remove the upgrade, it's not for 2.0 release.

Reply:
Removed. Good catch!